web/package.json

```diff
@@ -23,6 +23,7 @@
     "@mui/material": "^5.5.3",
     "@mui/styles": "^5.5.3",
     "@reduxjs/toolkit": "^1.8.0",
+    "@testing-library/user-event": "^14.1.1",
```

Comment:
Already exists in dev dependencies

Reply:
Oops! `yarn add` instead of `yarn add --dev` ; my bad!